adopt-a-dog/blog.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="zxx">
+<head>
+  <meta charset="utf-8">
+  <title>Adaptadog</title>
+  <!-- <link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet"> -->
+  <!-- <link rel="stylesheet" type="text/css" href="styles.css"> -->
+</head>
+<body>
+    <!-- <header> -->
+    <div class="container">
+      <img src="images/christopher-harris-57366-unsplash.jpg" alt="header image" height="700">
+      <button class="btn">Check Out</button>
+    </div>
+    <img src="images/cart-icon.svg" alt="shopping cart icon" height="40">
+    <h1>Adoptadog</h1>
+    <h2>Seattle's Premier Dog Adoption Agency</h2>
+    <nav>
+      <ul>
+        <li><a href="index.html">Home</a></li>
+        <li><a href="dogs.html">/Dogs</a></li>
+        <li><a href="blog.html">/Blog</a></li>
+      </ul>
+    </nav>
+
+  <h2>Adoptadog Blog</h2>
```

Comment:
off to a good start here. You might want to consider wrapping these in some sort of container for layout.